build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -131,7 +133,7 @@ spec:
           imagePullPolicy: {{ include "antreaAgentImagePullPolicy" . }}
           {{- if ((.Values.testing).coverage) }}
           command: ["/bin/sh"]
-          args: ["-c", "sleep 2; antrea-agent-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-agent.cov.out -args-file=/agent-arg-file; while true; do sleep 5 & wait $!; done"]
+          args: ["-c", "sleep 2; antrea-agent-coverage $(cat /agent-arg-file | tr '\n' ' '); while true; do sleep 5 & wait $!; done"]
```

Comment:
IMO, there is no good reason to keep `agent-arg-file` around any more. It was required with bincover, because there was no other way to pass actual command-line flags after wrapping the binary

Reply:
Should verbosity level be set to 4 if coverage is enabled? Or does it not matter?

Reply:
Whatever the current behavior is